src/omv/sensor.c

```diff
@@ -904,33 +904,43 @@ void DCMI_DMAConvCpltUser(uint32_t addr)
                 case PIXFORMAT_BAYER:
                     dst += (line - MAIN_FB()->y) * MAIN_FB()->w;
                     src += MAIN_FB()->x;
-                    for (int i = MAIN_FB()->w; i; i--) {
-                        *dst++ = *src++;
-                    }
+                    memcpy(dst, src, MAIN_FB()->w);
                     break;
                 case PIXFORMAT_GRAYSCALE:
                     dst += (line - MAIN_FB()->y) * MAIN_FB()->w;
+                    src += MAIN_FB()->x;
```

Comment:
Move under the if?

Reply:
This works correctly either as-is or after the if; I can change it for aesthetics.

Reply:
@bitbank2 yes please do so, it took me a while to figure out that it doesn't matter.

Reply:
done